### PR TITLE
Add `--version` option

### PR DESCRIPTION
**Pull request by mschnitzer**
* We need a `--version` option for this, because machinery needs to
  verify the version of the helper.

This is related to issue gh#SUSE/machinery#1340

### CODE REVIEW

**Comment by cornelius** [comment]
Does the release script still work with this change?

File: version.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package main
+
+const VERSION = "0.0.2"
```
Comment (cornelius) [inline version.go]: There is a newline missing at the end of the file.

**Comment by mschnitzer** [comment]
@cornelius yes it works but I changed some things in the release script. I'll do a PR for the release script.

**Comment by cornelius** [comment]
looks good to me

**Pull request merged**